miden-lib/asm/miden/kernel_proc_offsets.masm

```diff
@@ -1,96 +1,132 @@
 # OFFSET CONSTANTS
 # -------------------------------------------------------------------------------------------------
 
-# Current account
-const.ACCOUNT_VAULT_ADD_ASSET_OFFSET=0
-const.ACCOUNT_VAULT_GET_BALANCE_OFFSET=1
-const.ACCOUNT_VAULT_HAS_NON_FUNGIBLE_ASSET_OFFSET=2
-const.ACCOUNT_VAULT_REMOVE_ASSET_OFFSET=3
-const.GET_ACCOUNT_ID_OFFSET=4
-const.GET_ACCOUNT_ITEM_OFFSET=5
-const.GET_ACCOUNT_MAP_ITEM_OFFSET=6
-const.GET_ACCOUNT_NONCE_OFFSET=7
-const.GET_ACCOUNT_VAULT_COMMITMENT_OFFSET=8
-const.GET_CURRENT_ACCOUNT_HASH_OFFSET=9
-const.GET_INITIAL_ACCOUNT_HASH_OFFSET=10
-const.INCR_ACCOUNT_NONCE_OFFSET=11
-const.SET_ACCOUNT_CODE_OFFSET=12
-const.SET_ACCOUNT_ITEM_OFFSET=13
-const.SET_ACCOUNT_MAP_ITEM_OFFSET=14
-
-# Current faucet
-const.BURN_ASSET_OFFSET=15
-const.GET_FUNGIBLE_FAUCET_TOTAL_ISSUANCE_OFFSET=16
-const.MINT_ASSET_OFFSET=17
-
-# Current note
-const.ADD_ASSET_TO_NOTE_OFFSET=18
-const.CREATE_NOTE_OFFSET=19
-const.GET_INPUT_NOTES_COMMITMENT_OFFSET=20
-const.GET_NOTE_ASSETS_INFO_OFFSET=21
-const.GET_NOTE_INPUTS_HASH_OFFSET=22
-const.GET_NOTE_SENDER_OFFSET=23
-const.GET_NOTE_SERIAL_NUMBER_OFFSET=24
-const.GET_SCRIPT_HASH_OFFSET=25
-const.GET_OUTPUT_NOTES_COMMITMENT_OFFSET=26
-
-# Transaction
-const.GET_BLOCK_HASH_OFFSET=27
-const.GET_BLOCK_NUMBER_OFFSET=28
-const.START_FOREIGN_CONTEXT_OFFSET=29
-const.END_FOREIGN_CONTEXT_OFFSET=30
-const.UPDATE_EXPIRATION_BLOCK_NUM_OFFSET=31
-const.GET_EXPIRATION_DELTA_OFFSET=32
+### Account #####################################
+# Entire account hash
+const.ACCOUNT_GET_CURRENT_HASH_OFFSET=0
+const.ACCOUNT_GET_INITIAL_HASH_OFFSET=1
+
+# ID
+const.ACCOUNT_GET_ID_OFFSET=2
+
+# Nonce
+const.ACCOUNT_GET_NONCE_OFFSET=3                 # accessor
+const.ACCOUNT_INCR_NONCE_OFFSET=4                # mutator
+
+# Procedures
+## get account code is missing
+const.ACCOUNT_PROCEDURES_SET_CODE_OFFSET=5
+
+# Storage
+const.ACCOUNT_STORAGE_GET_ITEM_OFFSET=6
+const.ACCOUNT_STORAGE_SET_ITEM_OFFSET=7
+const.ACCOUNT_STORAGE_GET_MAP_ITEM_OFFSET=8
+const.ACCOUNT_STORAGE_SET_MAP_ITEM_OFFSET=9
+
+# Vault
+const.ACCOUNT_VAULT_ADD_ASSET_OFFSET=10
+const.ACCOUNT_VAULT_REMOVE_ASSET_OFFSET=11
+const.ACCOUNT_VAULT_GET_BALANCE_OFFSET=12
+const.ACCOUNT_VAULT_HAS_NON_FUNGIBLE_ASSET_OFFSET=13
+
+## set commitment is missing, but corresponding memory procedure is used in the
+## start_foreign_context kernel procedure
+const.ACCOUNT_VAULT_GET_COMMITMENT_OFFSET=14
+
+
+### Faucet ######################################
+const.FAUCET_BURN_ASSET_OFFSET=15
+const.FAUCET_FUNGIBLE_GET_TOTAL_ISSUANCE_OFFSET=16
+
+# add a new procedure to the kernel
+# const.FAUCET_NON_FUNGIBLE_IS_ISSUED_OFFSET=17
+const.FAUCET_MINT_ASSET_OFFSET=18
```

Comment:
Let's change the order to be:
```
const.FAUCET_MINT_ASSET_OFFSET=16
const.FAUCET_BURN_ASSET_OFFSET=17
const.FAUCET_GET_TOTAL_FUNGIBLE_ASSET_ISSUANCE_OFFSET=18
const.FAUCET_IS_NON_FUNGIBLE_ASSET_ISSUED_OFFSET=19
```

And let's add `FAUCET_IS_NON_FUNGIBLE_ASSET_ISSUED_OFFSET` (can stab it out for now and add the real implementation in a separate PR).